examples/all-clusters-app/esp32/README.md

```diff
@@ -207,11 +217,11 @@ commissioning and cluster control.
 ### Note
 
 This demo app illustrates controlling OnOff cluster (Server) attributes of an
-endpoint. For `ESP32-DevKitC` and `ESP32-WROVER-KIT_V4.1`, a GPIO (configurable
-through `STATUS_LED_GPIO_NUM` in `main/main.cpp`) is updated through the
-on/off/toggle commands from the `python-controller`. For `M5Stack`, a virtual
-Green LED on the display is used for the same.
-
-If you wish to see the actual effect of the commands on `ESP32-DevKitC` and
-`ESP32-WROVER-KIT_V4.1`, you will have to connect an external LED to GPIO
-`STATUS_LED_GPIO_NUM`.
+endpoint. For `ESP32-DevKitC`, `ESP32-WROVER-KIT_V4.1` and `ESP32C3-DevKitM`, a
+GPIO (configurable through `STATUS_LED_GPIO_NUM` in `main/main.cpp`) is updated
+through the on/off/toggle commands from the `python-controller`. For `M5Stack`,
+a virtual Green LED on the display is used for the same.
+
+If you wish to see the actual effect of the commands on `ESP32-DevKitC`,
+`ESP32-WROVER-KIT_V4.1` and `ESP32C3-DevKitM`, you will have to connect an
+external LED to GPIO `STATUS_LED_GPIO_NUM`.
```

Comment:
Is an on-board LED present on C3? If yes, the above is not required for it?

Reply:
The on-board LED is W2812(GPIO8) which is controlled by RMT. I am not sure this LED will works.